src/main/webapp/styles/main.css

```diff
@@ -0,0 +1,57 @@
+body {
```

Comment:
This page contains the basic styling points for all the design elements in the original PSD. This includes:

- The header section
- Horizontal navigation in the header
- Pink button
- Blue button
- Headings
- Paragraphs

More involved layout settings like margin and padding should be done when designing one of the Views, as it's difficult to gauge what it would look like prior to this stage.

I'm happy to offer support / cleanup when someone comes to this point in the work.